spec/models/book_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  subject(:book) { described_class.new(gender: gender, author: author, image: image, title: title, editor: editor, year: year) }
+
+  let(:gender) { Faker::Book.genre }
+  let(:author) { Faker::Book.author }
+  let(:image) { 'ImagenFalsa123.gif' }
+  let(:title) { Faker::Book.title }
+  let(:editor) { Faker::Book.publisher }
+  let(:year) { '2006' }
+
+  it do
+    is_expected.to be_valid
+  end
+
+  describe '#Create' do
+    context 'when the user is created' do
+      it 'also creates its random gender' do
```

Comment:
remove `also`

Reply:
Because, this is a way of documentation and when the test run it would said: `when the user is created also creates its random gender`

Reply:
I will no test the happy test that it works, i would only test if you are creating it with null values